.github/workflows/build.yml

```
@@ -93,9 +95,32 @@ jobs:
           name: ${{ steps.setup_precompiled.outputs.artifact_name }}
           path: ..
 
+  lint:
+    name: lint mathlib
```

```suggestion
    name: Lint mathlib
```

I don't particularly care one way or the other, but we already have "Build mathlib" and we should be consistent.